externs/shaka/player.js

```diff
@@ -1304,7 +1308,6 @@ shaka.extern.LcevcConfiguration;
  */
 shaka.extern.OfflineConfiguration;
 
-
```

Comment:
Remove this change

Reply:
Looks like spacing due to merge w LCEVC changes. Has been resolved.

Reply:
Please, remove this change